src/wasm.h

```diff
@@ -56,6 +56,13 @@
 
 namespace wasm {
 
+enum Features : uint32_t {
```

Comment:
32 bits should be enough for all the features? ;)

Reply:
Could just use a struct of bools... :-)

https://github.com/WebAssembly/wabt/blob/master/src/feature.h#L26

Reply:
Then we wouldn't be able to bitwise or them together like Real C Programmers should.... although if they are going to have names, then generating them from a def file like that might make sense. Maybe an improvement for the future.